dawn/src/dawn/Optimizer/PassSetBoundaryCondition.cpp

```diff
@@ -133,6 +128,10 @@ bool PassSetBoundaryCondition::run(
     }
     return true;
   }
+
+  iir::StencilMetaInformation& metadata = stencilInstantiation->getMetaData();
```

Comment:
this should not be necessary. It was a requirement to have happened before

Reply:
Afraid I do not understand, a test that no longer exists should not be a dependency, correct?